README.markdown

```diff
@@ -1,24 +1,26 @@
-# jQuery Standards Group
+# JS Foundation Standards Group
 
-The jQuery Standards Group exists to give web developers a voice in the
+*Note: this repository may contain historical references to the jQuery Standards Group. Prior to the fall of 2016, the JS Foundation was known as the jQuery Foundation. Its past participants and contributions are listed here in their historical contexts.*
```

Comment:
lol for some reason i read “the fall of 2016” like “the fall of the empire”

Reply:
hahaha yes, the year the empire of jQuery came tumbling down 😂